go/appencryption/pkg/persistence/persistencetest/dynamodb.go

```diff
@@ -0,0 +1,243 @@
+package persistencetest
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/docker/go-connections/nat"
+	"github.com/testcontainers/testcontainers-go"
+
+	"github.com/godaddy/asherah/go/appencryption"
+	"github.com/godaddy/asherah/go/appencryption/pkg/persistence"
+)
+
+const (
+	IKCreated          = int64(1541461381)
+	SKCreated          = int64(1541461380)
+	EncryptedKeyString = "mWT/x4RvIFVFE2BEYV1IB9FMM8sWN1sK6YN5bS2UyGR+9RSZVTvp/bcQ6PycW6kxYEqrpA+aV4u04jOr"
+	SKKeyID            = "_SK_api_ecomm"
+	TestKey            = "some_key"
+)
+
+const (
```

Comment:
maybe add a comment to indicate why constants are defined in 2 parts?

Reply:
There's nothing to comment on really as it's just a personal style preference. I find that separating exported consts from non-exports a bit cleaner. If you feel strongly otherwise I think I'd just do away with the second group and just leave a blank line in between.

Reply:
That fine, i can go either way.